lib/ifupdown.sh.in

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 
+# This script should be symlinked into /etc/network/ip-pre-up.d/ and
```

Comment:
Prior to this commit, ifupdown.sh had a few blocks that were formatted like bridge-stp, and had a lot of blocks that were formatted differently.  My primary goal with this commit was to make the ifupdown.sh formatting internally consistent - one way or the other.  Since bridge-stp already seemed to have consistent formatting, I thought it would be best to mimic that.  This formatting is actually not exactly the formatting I personally prefer to use in shell scripts ... so if you want to change the "preferred style" to something else, that is fine with me.  I was just trying to make it consistent.

There is a formatter here, although I haven't used it enough to say how well it works in practice: http://arachnoid.com/python/beautify_bash_program.html